### ISSUE TITLE
NVDA cannot read CLDR characters after #14459

### ISSUE THREAD

**Issue by OzancanKaratas** [nvaccess/nvda#14473]
### Steps to reproduce:
1. Use NVDA in your preferred language other than English.
2. Open the Windows Input Experience or visit emojipedia.org website.
3. Navigate in platform you selected.
### Actual behavior:
NVDA cannot read characters specified in CLDR.
### Expected behavior:
NVDA should read the characters as specified in CLDR.
### NVDA logs, crash dumps and other attachments:
None
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
All versions after #14459 merged.
#### Windows version:
Windows 11
#### Name and version of other software in use when reproducing the issue:
None
#### Other information about your system:
Not needed
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
Closed via #14477

**Comment by CyrilleB79** [comment]
I have not been able to reproduce this issue with French emojis before the revert in #14477 .

@OzancanKaratas  the steps to reproduce are a bit generalistic. Could you describe something more precise so that I can reproduce the issue?


@seanbudd could you confirm that you have reproduced the issue before reverting in #14477 ? If yes, could you help also with a more detailed description?

**Comment by seanbudd** [comment]
I did not reproduce the issue

**Comment by CyrilleB79** [comment]
> I have not been able to reproduce this issue with French emojis before the revert in #14477 .
> 
> @OzancanKaratas the steps to reproduce are a bit generalistic. Could you describe something more precise so that I can reproduce the issue?

@OzancanKaratas could you please help me reproduce this issue giving me more detailed steps? I will open a PR and Cc you so that we can test again. Thanks.

**Comment by OzancanKaratas** [comment]
NVDA was not reading anything while navigating emojis in Turkish. There was no problem before your pull request was merged and after it was reverted.
